### ISSUE TITLE
Doesn't work on iCloud folders?

### ISSUE THREAD

**Issue by austinarchibald** [sbarex/MediaInfo#29]
Had no problems getting this to work on internal, external folders, etc. But I cannot get it to work on iCloud folders. e.g. `/Users/austin/Library/Mobile Documents/com~apple~photos~dtp~photos/Documents/`

**Comment by sbarex** [comment]
yes, at the moment I can't get it to work on folders managed by other extensions (like iclod, dropbox, ...)
